src/lib/core/say.h

```diff
@@ -260,6 +260,10 @@ log_set_format(struct log *log, log_format_func_t format_func);
 void
 say_set_log_level(int new_level);
 
+/** Set log level for the on_log callback. */
+void
+say_set_on_log_level(int new_level);
```

Comment:
Since this is needed only if there's a callback, better pass it as an extra argument to `say_set_log_callback`.

Reply:
Ok, refactored.